app/models/conference.rb

```diff
@@ -328,9 +328,10 @@ def event_distribution
   # ====Returns
   # * +hash+ -> hash
   def scheduled_event_distribution
-    confirmed_events = program.events.where(state: 'confirmed')
-    scheduled_value =  { 'value' => confirmed_events.where.not(start_time: nil).count, 'color' => 'green' }
-    unscheduled_value =  { 'value' => confirmed_events.where(start_time: nil).count, 'color' => 'red' }
+    confirmed_scheduled_events = Event.scheduled.select{|event| event.program == program}
```

Comment:
You can apply the scope to the events returned by program so you don't need to add the select part :wink:

`confirmed_scheduled_events = program.events.scheduled`